pyproject.toml

```diff
@@ -46,7 +47,10 @@ bindings = "pyo3"
 
 [tool.pytest.ini_options]
 testpaths = ["tests"]
-filterwarnings = ["error"]
+filterwarnings = [
+    "error",
+    "ignore:The hookimpl CovPlugin:pytest.PytestDeprecationWarning",
```

Comment:
This is needed to eliminate deprecation warnings from pytest-cov:
```
pytest.PytestDeprecationWarning: The hookimpl CovPlugin.pytest_configure_node uses old-style configuration options (marks or attributes).
Please use the pytest.hookimpl(optionalhook=True) decorator instead
 to configure the hooks.
 See https://docs.pytest.org/en/latest/deprecations.html#configuring-hook-specs-impls-using-markers
 at /opt/hostedtoolcache/Python/3.11.7/x64/lib/python3.11/site-packages/pytest_cov/plugin.py:256

```

https://github.com/samuelcolvin/rtoml/actions/runs/7263394585/job/19788594082#step:9:81

Reply:
Note: upgrading pytest-cov from 3.0.0 to 4.1.0 does not help here.